app/code/core/Mage/Adminhtml/Block/System/Account/Edit/Form.php

```diff
@@ -97,6 +97,38 @@ protected function _prepareForm()
             'required' => false,
         ]);
 
+        $fieldset = $form->addFieldset('passkey_fieldset', [
+            'legend' => Mage::helper('adminhtml')->__('Passkey Authentication'),
+        ]);
+
+        if ($user->getPasskeyCredentialIdHash()) {
+            $fieldset->addField('passkey_status', 'note', [
+                'label' => Mage::helper('adminhtml')->__('Status'),
+                'text'  => '<span class="grid-severity-notice"><span>' .
+                    Mage::helper('adminhtml')->__('Passkey Registered') . '</span></span>',
+            ]);
+            $fieldset->addField('passkey_status', 'note', [
+                'text'  => $this->getButtonHtml($this->__('Remove Passkey'), null, 'delete', 'remove-passkey-btn'),
```

Comment:
I keep wondering how I don't know these basic stuff 🤣

Reply:
I knew there was a way, but had to grep to find the exact method.